install-dev/theme/views/process.php

```diff
@@ -64,27 +64,27 @@
 		<p><?php echo $this->translator->trans('Please remember your login information:', array(), 'Install'); ?></p>
 		<table cellpadding="0" cellspacing="0" border="0" id="resultInstall" width="620">
 			<tr class="odd">
-				<td class="label"><?php echo $this->translator->trans('E-mail', array(), 'Install') ?></td>
-				<td class="resultEnd"><?php echo htmlspecialchars($this->session->admin_email) ?></td>
-				<td rowspan="2" class="print" onclick="$('#password_content').text('<?php echo htmlspecialchars(addslashes($this->session->admin_password)) ?>'); $('#password_display').hide(); window.print();">
+				<td class="label"><?php echo $this->translator->trans('E-mail', array(), 'Install'); ?></td>
+				<td class="resultEnd"><?php echo htmlspecialchars($this->session->admin_email); ?></td>
+				<td rowspan="2" class="print" onclick="$('#password_content').text('<?php echo htmlspecialchars(addslashes($this->session->admin_password)); ?>'); $('#password_display').hide(); window.print();">
 					<img src="theme/img/print.png" alt="" style="vertical-align:top">
-					<?php echo $this->translator->trans('Print my login information', array(), 'Install') ?>
+					<?php echo $this->translator->trans('Print my login information', array(), 'Install'); ?>
```

Comment:
Actually, this seems pretty weird for me, offering new users to print their login information. Don't you think so? Isn't it dangerous regarding the use of personal data?

Reply:
Sure. How would you like the wording?

Reply:
Well, it is rather the behavior than the wording that concerns me. But we can leave it as-is since the installer will be reworked. @colinegin, @TristanLDD, what do you think?